test_rclcpp/test/test_allocator.cpp

```diff
@@ -34,9 +34,30 @@
 
 #endif  // not WIN32
 
+static const size_t num_rmw_tokens = 5;
+static const char * rmw_tokens[num_rmw_tokens] = {"librmw", "dds", "DDS", "dcps", "DCPS"};
```

Comment:
The rmw impl. packages would need to export that kind of information in order to be used here. That way other implemenations could do the same thing.

The same comment applies to a recent PR for launch: https://github.com/ros2/launch/pull/13#discussion_r44995670